inst/sympref.m

```diff
@@ -249,17 +230,6 @@
         settings.digits = arg;
       end
 
-    case 'snippet'
```

Comment:
How much do you care about deprecation vs deleting? Should sympref warn with "the snippet option has been removed"?